src/redux/uniqueTokens.ts

```diff
@@ -338,6 +339,17 @@ export const fetchUniqueTokens = (showcaseAddress?: string) => async (
     analytics.identify(null, { NFTs: uniqueTokens.length });
   }
 
+  // Fetch recently registered ENS tokens (OpenSea doesn't recognize these for a while).
+  // We will fetch tokens registered in the past hour to be safe.
+  const ensTokens = await fetchEnsTokens({
+    address: accountAddress,
+    timeAgo: { hours: 1 },
+  });
+  if (ensTokens.length > 0) {
+    uniqueTokens = uniqBy([...uniqueTokens, ...ensTokens], 'id');
```

Comment:
looking really awesome!!! 

from ibrahim's QA and myself, I have this

<img width="368" alt="image" src="https://user-images.githubusercontent.com/12115171/168903946-2060020a-46cc-42c8-b42b-7a8e28ecefe9.png">

i bought scamdog.eth yesterday, and opensea has not picked it up yet so I don't see any picture. When I put 48 hours in the fetch param in fetchEnsTokens, the ens is correctly in `ensTokens` with the right image but since `uniqueTokens` is first, `uniqBy` is taking the one with no `image_url`.

Could we also filter by image_url? like if there are ens repeated, let's pick up the one with image if there's no image coming from `uniqueTokens`

Reply:
Yeah sounds good. I think I'd rather fix the no image from OpenSea rather than picking up the one with the image (could be an edge case where there has been no registration in the past hour, but for some reason, OpenSea is being silly). I might make it fall back to the ENS metadata API if no image is coming back from OpenSea.